packages/node-sdk/src/client.ts

```diff
@@ -849,16 +903,24 @@ export class BucketClient {
     };
     checkContextWithTracking(contextWithTracking);
 
-    const params = new URLSearchParams(flattenJSON({ context }));
+    const params = new URLSearchParams(
+      Object.keys(context).length ? flattenJSON({ context }) : undefined,
```

Comment:
first line in `flattenJSON` is `if (Object.keys(data).length === 0) return {};`. This doesn't seem necessary?

Reply:
That's the issue actually. An empty object serializes as "[Object]" string which is not what we want.

Reply:
```
❯ node
Welcome to Node.js v18.16.0.
Type ".help" for more information.
> new URLSearchParams({}).toString()
''
>
```

Reply:
Maybe because it's an empty object within an empty object. I get: `"https:// api. example. com/ features/ evaluated?context=%5Bobject+Object%5D"`